### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by aoki**
- Change comment out sign `//` to `#` in shell script
- Add new line at end of file

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -54,14 +54,14 @@ $ mkdir tutorial
 $ cd tutorial
 $ mkdir rootfs
 $ docker export $(docker create busybox) | tar -C rootfs -xvf -
-// Prepare a configuration file for the container that will run `sleep 5`.
+$ # Prepare a configuration file for the container that will run `sleep 5`.
```
Comment (utam0k) [inline README.md:57]: There are only comments in this line, have you any reason to put `$` at the beginning?

**Comment by utam0k** [comment]
@aoki 
Hi, aoki.
Thank you for your PR. This is nice work. I have added some comments if you would like to see them.

File: README.md (hunk 1 above)
Comment (aoki) [reply README.md:57]: I think sometimes confuse `stdout` when not add `$` mark ahead (or execute on root `#`).

Another approach is to remove new line and join before command.
Like this, 
```sh
$ docker export $(docker create busybox) | tar -C rootfs -xvf -  # Prepare a configuration file for the container that will run `sleep 5`.
```
Which do you like? I think both okay.

Comment (utam0k) [reply README.md:57]: That's certainly a good perspective.
> I think sometimes confuse stdout when not add $ mark ahead (or execute on root #).

I think it's fine as it is now.  If you don't have time, this PR will be very good to merge as it is.
If you like, how about dividing some commands into meaningful units (e.g. build/prepare for container execution / execute container)? This way you could add a light description to each separated section to erase this commented-out description.

Comment (aoki) [reply README.md:57]: Thank you for your suggestion. Okay, I change to separate some section. Please wait a moment.

**Review by utam0k — APPROVED** [review]
lgtm

**Pull request merged**